services/common/src/redux/slices/complianceReportsSlice.ts

```diff
@@ -3,129 +3,197 @@ import { ENVIRONMENT } from "@mds/common/constants/environment";
+    },
+    getMineReportDefinitionOptions: (state): IMineReportDefinition[] => {
+      return state.reportPageData.records;
+    },
+    getComplianceReportPageData: (state): IPageData<IMineReportDefinition> => {
+      return state.reportPageData;
+    },
+    getReportSearchParams: (state): ComplianceReportParams => {
+      return state.params;
+    },
+    getMineReportDueDateTypes: (state): IMineReportDueDateType[] => {
+      return state.dueDateTypes;
+    },
+  },
 });
 
-export const { getMineReportDefinitionHash, getMineReportDefinitionOptions, getComplianceReportPageData, getReportSearchParams
+export const {
+  getMineReportDefinitionHash,
+  getMineReportDefinitionOptions,
+  getComplianceReportPageData,
+  getReportSearchParams,
+  getMineReportDueDateTypes,
 } = complianceReportSlice.selectors;
 
 export const getDropdownMineReportDefinitionOptions = createSelectorWrapper(
-    getMineReportDefinitionOptions,
-    createDropDownList, ["report_name", "mine_report_definition_guid", "active_ind"]
+  getMineReportDefinitionOptions,
+  createDropDownList,
+  ["report_name", "mine_report_definition_guid", "active_ind"]
 );
 
 export const getFormattedMineReportDefinitionOptions = createSelectorWrapper(
-    getMineReportDefinitionOptions,
-    (options: IMineReportDefinition[]) => {
-        return options
-            .map((item) => {
-                return {
-                    label: formatComplianceCodeReportName(item),
-                    value: item.mine_report_definition_guid,
-                    isActive: item.active_ind,
-                    is_common: item.is_common,
-                    report_name: item.report_name,
-                };
-            })
-            .sort((a, b) => a.label.localeCompare(b.label));
-    }
+  getMineReportDefinitionOptions,
+  (options: IMineReportDefinition[]) => {
+    return options
+      .map((item) => {
+        return {
+          label: formatComplianceCodeReportName(item),
+          value: item.mine_report_definition_guid,
+          isActive: item.active_ind,
+          is_common: item.is_common,
+          report_name: item.report_name,
+        };
+      })
+      .sort((a, b) => a.label.localeCompare(b.label));
+  }
 );
 
 export const getMineReportDefinitionByGuid = (mineReportDefinitionGuid: string) =>
-    createSelector([getMineReportDefinitionHash], (reportMap) => {
-        return reportMap[mineReportDefinitionGuid]
-    });
+  createSelector([getMineReportDefinitionHash], (reportMap) => {
+    return reportMap[mineReportDefinitionGuid];
+  });
 
 export const getReportDefinitionsLoaded = (params: ComplianceReportParams) =>
-    createSelector([getReportSearchParams], (currentParams) => {
-        return isEqual(params, currentParams);
-    })
-
-export const {
-    fetchComplianceReports
-} = complianceReportSlice.actions;
+  createSelector([getReportSearchParams], (currentParams) => {
+    return params === currentParams;
```

Comment:
this should be the lodash isEqual. I think some weird stuff happened with a merge resolve with this file, not sure why semicolons and stuff are being removed too.